Commonalities/documentation/CAMARA-AuthN-AuthZ-Concept.md

```diff
@@ -26,6 +26,12 @@ Authorization [2] refers to the process of verifying what a user has access to.
 
 It is also important to reflect on a step before authentication (step 0) - onboarding. This is also referred to commonly as partner/consumer onboarding. 
 
+### User indentity
+
+On what refers to CAMARA APIs to be exposed by Telco operators, the end user is the human participant which is identified in Telco Operator by a unique user identifier (e.g. Subject identifier `sub` in OIDC terminology). Therefore, the Authentication process allows to confirm/validate user identity.
+
+It is important to differentiate the identification of a user from the identification of a device or UE. For example, the same user could have multiple associated devices.  
+
```

Comment:
- With user, how do we differentiate between subscriber and the end user of the device and which one are we referring to here?

Reply:
Hi Shilpa, sorry for the late response. We are referring as `user` as the person who uses the Operator capabilities (end user of the device). But normally they will be the person who has a contract relationship with the Telco Operator (the one paying the bills) unless the operator have specific features to identify whether the end user using the device is not the one having the contract relationship with the Operator... which I guess you mean by subscriber.

Just to provide an example, in case of TEF in Brazil, there are family plans with multiple phone numbers. One "owner" phone number (the one corresponding to the user having the contract relationship) and other dependent phone numbers (used by other family members not being the owner). And we have the mechanisms in place to have different users for each phone number (different user id and profile). Otherwise, from Operator perspective the end user will be always the subscriber.

Reply:
Thanks for the clarification.